### PR TITLE
config for replication of revenue db to snowflake.raw.revenue_db_public

### PR DESCRIPTION
**Pull request by leanbc2**
- Update docker-compose.yaml
- Update .env
- Create a Readme for Us and store the original readme in `original_readme/README.md`
- Create a Make file in the root of the repo for disaster recovery, forward EC2 port and install octavia.
- Add current Octavia config

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -47,7 +47,7 @@ services:
       - POSTGRES_PASSWORD=${DATABASE_PASSWORD}
       - POSTGRES_USER=${DATABASE_USER}
     volumes:
-      - db:/var/lib/postgresql/data
+      - /data/db:/var/lib/postgresql/data
```
Comment (SardorQogita) [inline docker-compose.yaml]: hardcoding the path like this is a bad practice. Why can't we just use db volume?

File: docker-compose.yaml [hunk 2]
```diff
@@ -156,7 +156,6 @@ services:
       - 8001
     volumes:
       - workspace:${WORKSPACE_ROOT}
-      - data:${CONFIG_ROOT}
```
Comment (SardorQogita) [inline docker-compose.yaml:159]: Why are we removing the mount?

**Review by SardorQogita — CHANGES_REQUESTED** [review]

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+move_docker_compose_file_to_root_folder:
+	ssh ec2-user@$(SERVER) 'sudo cp ~/docker-compose.yaml /docker-compose.yaml';
+
+move_env_file_to_root_folder:
+	ssh ec2-user@$(SERVER) 'sudo cp ~/.env /.env';
+
+run_docker_compose_up:
+	ssh ec2-user@$(SERVER) "cd /; docker-compose up -d;";
+
+disaster_recovery: copy_docker_compose_to_ec2 copy_env_file move_docker_compose_file_to_root_folder \
```
Comment (SardorQogita) [inline Makefile:25]: How will the secrets recover?

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+	scp -i $(HOME)/.ssh/airbyte.pem ${DIR}/docker-compose.yaml ec2-user@ec2-18-132-194-109.eu-west-2.compute.amazonaws.com:~/docker-compose.yaml
+
+copy_env_file:
+	scp -i $(HOME)/.ssh/airbyte.pem ${DIR}/.env ec2-user@ec2-18-132-194-109.eu-west-2.compute.amazonaws.com:~/.env
+
+move_docker_compose_file_to_root_folder:
+	ssh ec2-user@$(SERVER) 'sudo cp ~/docker-compose.yaml /docker-compose.yaml';
+
+move_env_file_to_root_folder:
+	ssh ec2-user@$(SERVER) 'sudo cp ~/.env /.env';
```
Comment (SardorQogita) [inline Makefile]: I think we should use .env.<env_name> file instead of this template .env file

File: docker-compose.yaml (hunk 2 above)
Comment (leanbc2) [reply docker-compose.yaml:159]: the data volume is only needed for backward compatibility; when users upgrade from an old Airbyte version that relies on file-based configs

File: docker-compose.yaml (hunk 1 above)
Comment (leanbc2) [reply docker-compose.yaml]: Fixed

File: Makefile (hunk 3 above)
Comment (leanbc2) [reply Makefile:25]: They wont. Integrtion with secrets manager will happen next

**Review by SardorQogita — APPROVED** [review]

**Pull request closed without merging**